core/CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 ## 1.15.0
   - Update base python image to `python:3.11-alpine`
   - Update python dependacies
+  -  Support array result include sequence action (#92)
```

Comment:
Do we have to add a link to the PR explicitly just like [this](https://github.com/apache/openwhisk-runtime-docker/pull/101/files#diff-78487a5c5ddc3583bba367131aee9230c451649e5f39a2522a784effcf6d6c66R41)?

Reply:
If the PR is to the same repository as the release, then GitHub automatically converts the short form (#92) to the appropriate link when it is in the description of the release.  We only need the longer form when we want to link to a PR in a different repository.  Thanks for thinking of this though.

Reply:
Yes but it seems it's not working in the preview.
https://github.com/apache/openwhisk-runtime-docker/blob/master/core/CHANGELOG.md